app/dao/notifications_dao.py

```diff
@@ -201,20 +224,22 @@ def _update_notification_status(
             status,
         )
         current_app.logger.exception(e)
-        return notification
     except ArgumentError as e:
         current_app.logger.warning('Cannot update notification %s to status %s', notification.id, status)
         current_app.logger.exception(e)
-        return notification
-    except:
+    except Exception:
```

Comment:
This will work, but the change isn't necessary.  I only use "Exception" when I need to reference the Exception instance.  i.e. `except Exception as e:`.

Reply:
ruff complains about bare exception cases, adding Exception here because of that.